app/controllers/texts_controller.rb

```diff
@@ -1,4 +1,11 @@
 class TextsController < ApplicationController
   def index
+    if params[:genre] == nil
+      @texts = Text.where(genre: ["Basic", "Git", "Ruby", "Ruby on Rails"])
+    else
+      @texts = Text.where(genre: params[:genre])
+    end
```

Comment:
これでも一応動作しますが，想定外のジャンルを表示される可能性があるので，ホワイトリストにした方がよいですよ。（例えば `invisible` が表示されてしまいます）

`*****` の箇所は考えてください。
```suggestion
    if params[:genre].in?(*****)
      @texts = Text.where(genre: params[:genre])
    else
      @texts = Text.where(genre: ["Basic", "Git", "Ruby", "Ruby on Rails"])
    end
```

Reply:
自分で調べてみても何を()の中に代入すればいいのかわかりません。（.in?やホワイトリストだったり、やんばるエキスパートの教材でなにか該当する箇所がないか調べました。）

ホワイトリストという単語からストロングパラメーターを使うのかと思ったのですが、今回の課題はユーザーから入力する箇所ではないと思ったので、違うのかな等、そこの部分で悩んでいます。なにかヒントをいただけますでしょうか？